policies/coding-style.md

```diff
@@ -591,21 +591,66 @@ should be used. This is more commonly referred to as C90. ISO/IEC 9899:1999
 (also known as C99) is not supported on some platforms that OpenSSL is
 used on and therefore should be avoided.
 
-## Chapter 15: Miscellaneous
+## Chapter 15: Expressions
 
-Do not use `!` to check if a pointer is NULL, or to see if a str...cmp
-function found a match.  For example, these are wrong:
+Avoid needless parentheses.<br/>
+For example, do not write
```

Comment:
Normally a new paragraph is used in text. A blank line achieves that. `<br>` is normally only used in poems, postal addresses and such.

In this case, maybe no new paragraph is needed though.

```suggestion
Avoid needless parentheses. For example, do not write
```

Reply:
Right. I tend to add explicit newlines when I feel this make the text more clear/readable.

Reply:
ok. I think it looks like a typographic error mostly. In a paper or book, a new paragraph usually looks like a newline with some indentation on the next line, while on the web it's almost always vertical spacing. It's still a new paragraph in both cases though, just different rendering.

Reply:
All right, removed all the `<br/>`.